src/components/RequestGroupComposition/Configuration.vue

```diff
@@ -244,6 +414,26 @@ export default {
         return {};
       }
     },
+    ditherPatternOptions: {
```

Comment:
These props show up in a few different components because they need to be passed down from the main form component down to where they are needed in the configuration/instrument config components